components/to_shmem_derive/Cargo.toml

```diff
@@ -14,5 +14,5 @@ darling = "0.9"
 derive_common = { path = "../derive_common" }
 proc-macro2 = "0.4"
 quote = "0.6"
-syn = { version = "0.15", features = ["visit"] }
```

Comment:
Have you checked (for example in the HTML report for `-Z timings`) that this actually disables some feature? Since Cargo takes the union of features requested by all dependent crates, this could have no effect at all. And conversely, if this patch is wrong the build might not break.

Reply:
> Have you checked (for example in the HTML report for `-Z timings`) that this actually disables some feature? Since Cargo takes the union of features requested by all dependent crates, this could have no effect at all. And conversely, if this patch is wrong the build might not break.

Some other crates actually need `visit` in the graph, I just know `to_shmem_derive` doesn't need it (through `git grep -w visit components/to_shmem_derive`).